src/pages/pu-trx/[functionId]/index.js

```diff
@@ -0,0 +1,265 @@
+import { Grow } from '@mui/material'
+import { useRouter } from 'next/router'
+import React, { useContext } from 'react'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import RPBGridToolbar from 'src/components/Shared/RPBGridToolbar'
+import { useError } from 'src/error'
+import { useDocumentTypeProxy } from 'src/hooks/documentReferenceBehaviors'
+import { ControlContext } from 'src/providers/ControlContext'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { PurchaseRepository } from 'src/repositories/PurchaseRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { SystemFunction } from 'src/resources/SystemFunction'
+import { useWindow } from 'src/windows'
+
+// import SaleTransactionForm from './forms/SaleTransactionForm'
```

Comment:
clear this

Reply:
This is Imtithal's task, I’ve merged only from her branch. For now, ignore everything related to purchase